.pre-commit-config.yaml

```diff
@@ -2,7 +2,7 @@
     # See https://pre-commit.com/hooks.html for more hooks
     repos:
       - repo: https://github.com/ambv/black
-        rev: 23.3.0
+        rev: 23.12.1
         hooks:
           - id: black
-            language_version: python3.9
+            language_version: python3.12
```

Comment:
Is it okay if we settle on 3.11 for the bookworm release?  Not sure what new stuff is in 3.12, but we'd have to roll our own for the Raspberry Pi OS

Reply:
yes, 3.11 should be fine

Reply:
rolled this back